.github/workflows/CI.yml

```diff
@@ -3,7 +3,7 @@ name: CI
 env:
   ROLLBAR_ENV: 'GitHubCI'
 
-on: [push]
+on: [pull_request]
```

Comment:
needed the pull_request context here to differentiate from push-by-push changes. I'm not sure what other impact this might have @jnaviask

Reply:
This may be closer to what we want, so it still runs on pushes to master. Do we need it to run on any other branch pushes?
```
on:
  pull_request:
  push:
    branches:
      - master
```

Reply:
Master should be fine! As long as it runs on PR pushes.